content/authoring/kata/create-first-kata.md

```diff
@@ -0,0 +1,256 @@
+
+However, authors sometimes do not realize the fact that creating a good quality kata is much harder than solving it. Creating kata is a totally different kind of task from solving kata. It requires a much wider mindset and set of skills. Creating a kata is much closer to "professional" tasks: it requires you to design a task with a large set of target users in mind, write a solution, create tests for it, handle feedback from users, and maintain it by fixing bugs. Every kata is a small software product in itself, and goes through a small equivalent of a full software development lifecycle!
+
+
+## Support for authors
+
+To support you with this difficult task, a set of help pages has been created with the following types of information:
+- [tutorials](/tutorials/), for users who are still figuring things out,
+- [guidelines](/authoring/guidelines/), which need to be respected to meet the Codewars quality criteria. It is strongly recommended to become familiar with these, otherwise you risk that your kata will be met with bad reception, harsh remarks, and many reported issues.
+- [HOWTOs](/recipes/) explaining how to realize some commonly occurring tasks, or solve commonly repeating problems,
+- [Language specific pages](/languages/) with code snippets, examples, references, and tutorials related to specific programming languages.
+
+You can also reach out directly to the community to ask questions and seek experienced users' advice on the [kata-authoring-help](https://gitter.im/Codewars/codewars.com/kata-authoring-help) Gitter channel.
+
+
+## Requirements
+
+To create a new kata, you need to earn the ["Create Kata" privilege](/references/gamification/privileges/). After reaching the required amount of 300 Honor points, the privilege is granted and you can select the **"New Kata"** option from the profile menu:
+
+<div class="block dark:hidden">
+
+!["New Kata" menu command](./img/new_kata-light.png)
+
+</div>
+<div class="hidden dark:block">
+
+!["New Kata" menu command](./img/new_kata-dark.png)
+
+</div>
+
+
+
+## Kata Idea
+
+At the time of writing, there are already **over 6500** approved kata, and almost 2500 beta kata. It's a vast red ocean out there - if you're still sticking to ideas like Fizzbuzz, Fibonacci numbers, or Caesar/ROT cipher, it's pretty much `99.999999...%` chance that someone has done this before you.
+
+This is bad because creating a kata about them again constitutes a **duplicate** kata, and we certainly don't want 100 Fizzbuzz kata out in the wild. When a duplicate is found it will be retired, which basically means it gets taken out of the kata catalog.
+
+
+### How to not step on the minefield of duplicate kata
+
+
+#### Search for keywords to see if someone has done the idea before
+
+As we all know, before you even engage your enemies, you need to send scouts forward.
```

Comment:
Let's get rid of this "enemies" references in this section.

Reply:
I think I removed all.